server/src/main/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeAggregator.java

```diff
@@ -155,6 +156,11 @@ protected void doClose() {
         }
     }
 
+    @Override
+    public ScoreMode scoreMode() {
```

Comment:
I know this is wrong. I had to add this for the competitive iterator to be picked up. I need some guidance on what this method should return.